lepton/package.go

```diff
@@ -250,9 +250,10 @@ func GetLocalPackageList() ([]Package, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	username := GetLocalUsername()
 	for _, pkg := range localPackages {
 		pkgName := pkg.Name()
+		pkgParts := strings.Split(pkg.Name(), "-")
```

Comment:
as `from-docker` uses a `-` to join the image name and version, I felt that `-` would be appropriate to split the version

Reply:
this is better but this is only for the case where you import 'from-docker' - any package not imported via that (eg: you create yourself or create from a directory),

so all 3 of these are 'valid' and produce:

python_3.8.6 -> 3.8.6
node-16.3.0 -> 16.3.0
nginx -> ''

Reply:
ok so how would we handle `java_1.8.0_190` as we have such posibility. maybe be consider everything after the first `_` as version?

Reply:
ok maybe I can regex it

Reply:
yes for now just use the first underscore - looking towards the future this probably doesn't belong in the actual filename itself

Reply:
done

Reply:
so this is still populating packages without versions the name of the package which doesn't make sense - i'd just leave it empty string if there is no version;

also i have a package named 'node-16.3.0' and that is populating the version as 'node-16.3.0'

Reply:
in general a version really shouldn't have anything but numerics && periods